notebooks/src/el-paso-workflow.ipynb

```diff
@@ -284,26 +283,52 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Model V2: Decomposing the ventilation question"
+    "# Model V2: Decomposing the ventilation question"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "My strategy for this question will be to break it into pieces and then improve my estimate for each piece. I'll start by estimating the peak ventilators needed as the product of two things:\n",
+    "## My decomposition\n",
+    "\n",
+    "My strategy for this question will be to break it into pieces and then improve my estimate for each piece. I'll start by estimating the peak ventilators needed as the product of:\n",
     "\n",
-    "1. How many patients will be in the ICU on peak day?\n",
+    "1. How many patients will be in the ICU on the day of peak hospitalizations?\n",
     "2. What fraction of those will need ventilators?\n",
     "\n",
+    "So my decomposition looks like this:\n",
+    "\n",
+    "- \\# ventilators needed on peak day =\n",
+    "  - \\# icu patients on peak day *\n",
+    "  - % of icu patients requiring ventilation\n",
+    " \n",
     "For both of these I'll make pretty uninformed guesses for now:\n",
     "\n",
     "1. I guess that about 5 to 200 people will be in the ICU on peak day (but I have no idea)\n",
     "2. I'd guess that 1 out of 3 will need ventilators (but I have no idea)\n",
     "\n",
-    "For ICU cases, I use the same `lognormal_from_interval` function I used above.\n",
+    "For ICU cases, I use the same `lognormal_from_interval` function I used above. I'll always show components of my model in blue and the overall model prediction in red:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def peak_icu_patients():\n",
+    "    \"# icu patients on peak day\"\n",
+    "    return ergo.lognormal_from_interval(5, 200)\n",
     "\n",
-    "For ventilation need, I'll use a beta-binomial distribution:"
+    "show_marginal(peak_icu_patients)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For ventilation need, I'll use a [beta-binomial distribution](http://varianceexplained.org/statistics/beta_distribution_and_baseball/):"
```

Comment:
This is the explanation of the beta distribution that most helped me understand beta_from_hits